docs/templates/relation.html

```diff
@@ -1,4 +1,4 @@
-<article class="relation">
+<article class="relation" id="card">
```

Comment:
Why does this element need an id? I don't see anything referencing it (unless it was intended for the `.card` CSS selector, but that's for a class and not an id). But also, there should not be a constant id value in a template like this. An id should be unique in an HTML document, and this id `card` will get repeated for each `<article>` element.